index.html

```diff
@@ -33,28 +53,206 @@
         don't overflow the page width
       */
       img {
-        width: 100%;
+        width: 50%;
         display: block;
         margin: 0 auto;
+        padding: 0;
+        max-width: 30%;
+        max-height: 100%;
+        height: auto;
+      }
+
+      .li1 {
```

Comment:
Just a recommendation here - use classnames that are descriptive of what you are trying to do